core/store/migrations/migration1560924400/migrate.go

```diff
@@ -0,0 +1,39 @@
+package migration1560924400
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+)
+
+// Migrate adds the BlockHash column to the RunRequest and TxReceipt tables
+func Migrate(tx *gorm.DB) error {
+	if err := tx.AutoMigrate(&RunRequest{}).Error; err != nil {
```

Comment:
While it's great that we capture and isolate the type here from future modification, I believe this is also needed in `migration0` to retroactively capture what was true previous to this commit.

So we want `RunRequest` and `TxReceipt` types to be captures in migration0 as they were before this commit. Examples are in the migration 0 file:

migration0.go
```
...
// TxAttempt is a capture of the model representing TxAttempts before migration1559081901
type TxAttempt struct {
	Hash      common.Hash `gorm:"primary_key;not null"`
	TxID      uint64      `gorm:"index"`
	GasPrice  *models.Big `gorm:"type:varchar(255)"`
	Confirmed bool
	Hex       string `gorm:"type:text"`
	SentAt    uint64
	CreatedAt time.Time `gorm:"index"`
}
```